src/controllers/apps/auth/user.controllers.js

```diff
@@ -339,9 +339,7 @@ const forgotPasswordRequest = asyncHandler(async (req, res) => {
       // ! NOTE: Following link should be the link of the frontend page responsible to request password reset
       // ! Frontend will send the below token with the new password in the request body to the backend reset password endpoint
       // * Ideally take the url from the .env file which should be teh url of the frontend
-      `${req.protocol}://${req.get(
-        "host"
-      )}/api/v1/users/reset-password/${unHashedToken}`
+      `${process.env.FORGOT_PASSWORD_REDIRECT_URL}/${unHashedToken}`
```

Comment:
Please update the `.env.sample` file with newly added env var with necessary comment to understand what env var does.

Reply:
Updated .env.sample with the comments.